src/adapter/src/coord/ddl.rs

```diff
@@ -334,6 +335,34 @@ impl Coordinator {
                     update_default_arrangement_merge_options = true;
                     update_http_config = true;
                 }
+                catalog::Op::AlterSource { id, .. } | catalog::Op::RenameItem { id, .. } => {
+                    let item = self.catalog().get_entry(id);
+                    let is_webhook_source = item
+                        .source()
+                        .map(|s| matches!(s.data_source, DataSourceDesc::Webhook { .. }))
+                        .unwrap_or(false);
+                    if is_webhook_source {
+                        webhook_sources_to_restart.insert(*id);
+                    }
+                }
+                catalog::Op::RenameSchema {
```

Comment:
Does drop schema do the correct thing somehow through some other code path?

Reply:
AFAIU if a user drops the schema they'd need to also drop the webhook via `CASCADE`, which should generate an `op::DropObject`?